docs/prompt_engineering/how_to_guides/playground/custom_openai_compliant_model.mdx

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 4
+---
+
+# Use a custom OpenAI compliant model
+
+The LangSmith playground allows you to use any model that is compliant with the OpenAI API. You can utilize your model by setting the Proxy Provider for `OpenAI` in the playground.
```

Comment:
We should break this into two sections. 

Step 1: accept openai compliant format in your model or system. This just links to the docs of what they need to set up
Step 2: the instructions in playground

Reply:
fixed